src/components/Timeline/updates/TimelineJourneyStart.tsx

```diff
@@ -23,7 +23,10 @@ const TimelineJourneyStart: React.FC<TimelineJourneyStartProps> = ({
       }
       update={update}
     >
-      <ZetkinJourneyInstanceCard instance={update.details.data} />
+      <ZetkinJourneyInstanceCard
+        instance={update.details.data}
+        orgId={update.organization.id}
+      />
```

Comment:
Why not just pass in the entire `update` object, and use whatever you like inside? Just a thought!

Reply:
That would make sense in another situation I guess, but the `ZetkinJourneyInstanceCard` (or at least not the wrapped `ZetkinJourneyInstanceItem` which defines the properties) isn't just used for timeline updates, but also elsewhere (i.e. the new section on the profile page introduced in this PR).